bin/hdfcoinc/pycbc_multiifo_coinc_findtrigs

```diff
@@ -330,10 +330,8 @@ if len(data['stat']) > 0:
                          compression_opts=9,
                          shuffle=True)
 
-f['segments/coinc/start'], f['segments/coinc/end'] = veto.segments_to_start_end(coinc_segs)
-
-for t in trigs:
-    f['segments/%s/start' % trigs[t].ifo], f['segments/%s/end' % trigs[t].ifo] = trigs[t].valid
+# Store coinc segments keyed by detector combination
+f['segments/%s/start' % ''.join(sorted(detectors))], f['segments/%s/end' % ''.join(sorted(detectors))] = trigs[args.pivot_ifo].valid
```

Comment:
Does detectors need to be sorted? or should it come from the order on the command line, (the order of which comes from the precedence list I think)

Reply:
Having spoken with Tom, this should be kept as sorted alphabetically, in case the pivot/fixed and precedence scheme changes

Reply:
btw this line is too long, please split (PEP8!)

Reply:
Done